deploy/000-psr.ts

```diff
@@ -12,11 +12,11 @@ const func: DeployFunction = async ({
   const { deploy } = deployments;
   const { deployer } = await getNamedAccounts();
 
-  const vBNBAddress = (await ethers.getContractOrNull("vBNB"))?.address || ADDRESS_ONE;
-  const comptrollerAddress = (await ethers.getContractOrNull("Unitroller"))?.address || ADDRESS_ONE;
-  const WBNBAddress = (await ethers.getContractOrNull("WBNB"))?.address || ADDRESS_ONE;
-  const timelockAddress = (await ethers.getContractOrNull("NormalTimelock"))?.address || multisigs[name];
-  const acmAddress = (await ethers.getContractOrNull("AccessControlManager"))?.address || ADDRESS_ONE;
+  const vBNBAddress = (await ethers.getContract("vBNB"))?.address;
```

Comment:
The PSR contract is deployed to every network, but vBNB only exists on BNB chain. Moreover, the vBNB address cannot be the zero address in the PSR (checked by the contract). The workaround was to set the address one for vBNB in that case.

We are following the same pattern for WBNB (and for other contracts, like the Shortfall contract in the RiskFund)